h2o-algos/src/main/java/hex/gam/MatrixFrameUtils/GamUtils.java

```diff
@@ -86,7 +86,7 @@ public static void copy2DArray(double[][] src_array, double[][] dest_array) {
   }
 
   public static GLMParameters copyGAMParams2GLMParams(GAMParameters parms, Frame trainData, Frame valid) {
-    GLMParameters glmParam = new GLMParameters();
+    GLMParameters glmParam = new GLMParameters(GLMParameters.Family.gaussian, GLMParameters.Link.family_default);
```

Comment:
Is this necessary ? GLM receive whatever is set to GAM, be it GAM's default or custom value. We should ask @wendycwong if this should be the default for GAM as well.

Reply:
@koniecsveta @Pscheidl : I looked at GLMParameters constructor without any arguments:

    public GLMParameters() {
      this(Family.gaussian, Link.family_default);
      assert _link == Link.family_default;
      _stopping_rounds = 3;
      _stopping_metric = ScoreKeeper.StoppingMetric.deviance;
      _stopping_tolerance = 1e-4;
    }

It already sets to Gaussian family.  However, now I am wondering if this should be changed to family AUTO?

I agree with @Pscheidl that there is no need to add arguments to new GLMParameters.  I also agree with @koniecsveta changing the constructor to GLMParameters without parameters to what she did below.